tools/ci-ts/test-helpers/__snapshots__/common.test.ts.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`test-helpers/common #logEvents event callback event handler filters by event 1`] = `
```

Comment:
How about we use inlined snapshots? Makes it a lot easier to read the tests themselves. I typically use non-inline snapshots when the diffs are very big, but these are pretty small